File: dev/requirements/screen/cases/state-key.4.3.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 4.3 — the key naming the three card states.
let stateKey_4_3 = RequirementCase(id: "4.3") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "state-key", id: "4.3")
    }
}
#endif

File: dev/requirements/screen/cases/state-key.4.3.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 4.3 — A key names the three states, in order.
final stateKey_4_3 = RequirementCase('4.3', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.setListKey,
    slug: 'state-key',
    id: '4.3',
  );
});
